test/hotspot/jtreg/ProblemList-zgc.txt

```diff
@@ -41,6 +41,10 @@ serviceability/sa/ClhsdbPstack.java#id1                       8268722   macosx-x
 serviceability/sa/TestJmapCore.java                           8268722,8268283   macosx-x64,linux-aarch64
 serviceability/sa/TestJmapCoreMetaspace.java                  8268722,8268636   generic-all
 
+serviceability/sa/TestJhsdbJstackMixed.java                   8248912   generic-all
+serviceability/sa/ClhsdbPstack.java#id0                       8248912   generic-all
+serviceability/sa/ClhsdbPstack.java#id1                       8248912   generic-all
```

Comment:
I'm only seeing failures for 8248912 on linux-x64. You might
want to limit it to that platform.

Reply:
I considered it, but given it's ZGC, and the SA support for it is not that stable, I thought it best just to silence it on all platforms.

Reply:
Sounds like a reasonable plan. Thumbs up.